### ISSUE TITLE
alt+f not working when the cursor is on an input element

### ISSUE THREAD

**Issue by noelpur** [philc/vimium#2186]
This hotkey is handy and would be cool to have an option to keep it active even if the cursor is inside an input element.

**Comment by gdh1995** [comment]
On Windows, <Alt-F> is used by Chrome to open the main menu on the top right corner. So it's not a good idea.

**Comment by noelpur** [comment]
You are right, but I'm not particular about the key combination. I just want to be able to activate the hotkeys even if the cursor is inside an input element.

**Comment by smblott-github** [comment]
We try to avoid adding too many options to the options page.

This idea could be confusing for users.  Currently, either Vimium is handling keys or it is not.  Also, who knows what sites/extensions this idea would break?
